tests/api/test_parameters.py

```diff
@@ -280,11 +290,13 @@ def test_nullable_parameter(simple_app):
         '/v1.0/nullable-parameters?time_start={}'.format(time_start))
     assert json.loads(resp.data.decode('utf-8', 'replace')) == time_start
 
-    resp = app_client.post('/v1.0/nullable-parameters', data={"post_param": 'None'})
-    assert json.loads(resp.data.decode('utf-8', 'replace')) == 'it was None'
+    # XXX this has not valid OAS3 equivalent as far as I can tell
```

Comment:
I could use help figuring out the correct behavior here.
It seems like there should be something that iterates through form parameters and replaces `"null"` or `"None"` with `None`? Does this need to be recursive? Is it only for form encoded request bodies? Can form parameters have recursive definitions, or are they guaranteed to be flat?